sdk/communication/azure-communication-chat/test-resources.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
Similar to azure-communication-email’s existing[ test-resource.json](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/communication/azure-communication-email/test-resources.json), adding one for azure-communication-chat (overriding the [one](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/communication/test-resources) in service level.)